.circleci/config.yml

```diff
@@ -119,42 +119,42 @@ jobs:
 
   PyTorch-v1_1:
     docker:
-      - image: circleci/python:3.6
+      - image: pytorchlightning/pytorch_lightning:devel-pt1.1
     environment:
       - TORCH_VERSION: "torch>=1.1, <1.2"
     steps: *steps
 
   PyTorch-v1_2:
     docker:
-      - image: circleci/python:3.6
+      - image: pytorchlightning/pytorch_lightning:devel-pt1.2
     environment:
       - TORCH_VERSION: "torch>=1.2, <1.3"
     steps: *steps
 
   PyTorch-v1_3:
     docker:
-      - image: circleci/python:3.6
+      - image: pytorchlightning/pytorch_lightning:devel-pt1.3
     environment:
       - TORCH_VERSION: "torch>=1.3, <1.4"
     steps: *steps
 
   PyTorch-v1_4:
     docker:
-      - image: circleci/python:3.6
+      - image: pytorchlightning/pytorch_lightning:devel-pt1.4
     environment:
       - TORCH_VERSION: "torch>=1.4, <1.5"
     steps: *steps
 
   PyTorch-v1_5:
     docker:
-      - image: circleci/python:3.6
+      - image: pytorchlightning/pytorch_lightning:devel-pt1.5
     environment:
       - TORCH_VERSION: "torch>=1.5, <1.6"
     steps: *steps
 
   Examples:
     docker:
-      - image: circleci/python:3.7
+      - image: pytorchlightning/pytorch_lightning:devel-pt1.4
```

Comment:
Just a question: maybe we should separate the development image from the user image b y name and not only by tag? That we have something like `pytorch_lightning_devel:pt1.4`?

Reply:
I was checking on it and the image name has to be the same as repo name...
so not sure if we can create Drocker repo without existing real GitHub repo
so that's why I placed the devel to tag and it seems to be a common case like you see in the base image... adding `devel`/`running`